source/_integrations/plugwise.markdown

```diff
@@ -89,151 +89,118 @@ This integration follows standard integration removal. No extra steps are requir
-    sequence:
-      - action: select.select_option
-        target:
-          entity_id: select.bios_thermostat_schedule
-        data:
-          option: "Regulier"
-```
+- `off`: Regulation is set to off – The HVAC system does not heat or cool, but the domestic hot water heating function (if available) remains active.
 
-#### Change boiler setpoint
+If you have an Anna with Elga:
 
-action: `number.set_value`
+- `heat_cool`: No active schedule – The system is in automatic cooling or heating mode, using the active preset or manually set temperature to control the HVAC system.
 
-```yaml
-# Example script change the boiler setpoint
-script:
-  change_max_boiler_tempeture_setpoint:
-    sequence:
-      - action: number.set_value
-        target:
-          entity_id: number.opentherm_max_boiler_temperature_setpoint
-        data:
-          value: 60
-```
+{% note %}
+The last schedule that was active is determined the same way long-tapping the top of an Anna works.
+{% endnote %}
 
-#### Set temperature
+## Examples
 
-action: `climate.set_temperature`
+### Presence-based preset mode
 
-Example:
+Automation to change the active preset. The preset will be changed by your active schedule, or it could be a likewise automation setting another preset for when someone is present.
 
 ```yaml
-# Example script change the temperature
-script:
-  anna_set_predefined_temperature:
-    sequence:
-      - action: climate.set_temperature
-        target:
-          entity_id: climate.anna
-        data:
-          temperature: 19.5
+alias: "Set climate to away when nobody is home"
+triggers:
+  # When either occupant leaves for more than 15 minutes
+  - trigger: state
+    entity_id:
+      - person.mom
+      - person.dad
+    to: not_home
+    for:
+      minutes: 15
+conditions:
+  # If Anna is using the normal "home" preset
+  - condition: state
+    entity_id: climate.anna
+    attribute: preset_mode
+    state: home
+  # And nobody is home
+  - condition: state
+    entity_id: person.mom
+    entity_id: person.dad
+    state: not_home
+actions:
+  # Change Anna to Away
+  - action: climate.set_preset_mode
+    data:
+      preset_mode: away
+    target:
+      entity_id: climate.anna
 ```
 
-#### Set preset mode
-
-action: `climate.set_preset_mode`
-
-Available options include: `home`, `vacation` (Anna only), `no_frost`, `asleep` & `away`.
-
-Example:
+### Seasonal change of HVAC mode
 
 ```yaml
-# Example script changing the active (or currently set by schedule) preset
-script:
-  anna_activate_preset_asleep:
-    sequence:
-      - action: climate.set_preset_mode
-        data:
-          preset_mode: asleep
+alias: "Force Anna to cooling mode only"
+triggers:
+  # When the season changes to summer
+  - trigger: state
+    entity_id:
+      - sensor.season
+    to: summer
+conditions:
+  # If Anna is in either auto or heat mode
+  - condition: or
+    conditions:
+      - condition: device
+        device_id: DEVICE_ID_REPLACE_ME
```

Comment:
Let's avoid the use of device automations here, and use entity based conditions instead.

Reply:
Neither me nor @bouwew seem to be able to determine hvac_mode without leveraging the device, unless we don't want conditions on it?

Reply:
@CoMPaTech sorry, this automation makes no sense to me :slightly_smiling_face: 

The only thing the user needs to do is to switch the heat pump connected to the Anna or Adam into cooling-mode, or in case of the Anna+Elga, if configured for automatic switching (to cooling and back), nothing. And the Integration will automatically pick up the relevant changes.

Reply:
Re-worked the entire example section now including two most relevant blueprints

Reply:
If either of you can review the modified examples and links to the blueprints that would be highly appreciated